apps/website/docs/api/operators/keep_fresh.md

```diff
@@ -14,6 +14,7 @@ Config fields:
 
 - `automatically?`: _true_ to refresh the data in a [_Query_](/api/primitives/query) automatically if any [_Store_](https://effector.dev/docs/api/effector/store) that is used in the [_Query_](/api/primitives/query) creation is changed.
 - `triggers?`: _Array_ of [_Events_](https://effector.dev/docs/api/effector/event) after which operator starts refreshing the data in the [_Query_](/api/primitives/query).
+- `enabled?`: [_Store_](https://effector.dev/docs/api/effector/store) with the current enabled state. Disabled _keepFresh_ will not execute queries, instead, they will be treated as skipped. Can be `true` or `false`.
```

Comment:
🤓 could you add a badge for marking this option as available `since v0.11`? Examples available in the repo.

Reply:
Done!